src/platform/EFR32/BLEManagerImpl.cpp

```diff
@@ -723,6 +702,11 @@ CHIP_ERROR BLEManagerImpl::ConfigureAdvertisingData(void)
     responseData[index++] = ShortUUID_CHIPoBLEService[0]; // AD value
     responseData[index++] = ShortUUID_CHIPoBLEService[1];
 
+    responseData[index++] = static_cast<uint8_t>(mDeviceNameLength + 1); // length
+    responseData[index++] = CHIP_ADV_DATA_TYPE_NAME;                     // AD type : name
+    memcpy(&responseData[index], mDeviceName, mDeviceNameLength);        // AD value
```

Comment:
So this is enough to set the BleName here from this memcpy ?

Reply:
static_assert(kMaxDeviceNameLength <= UINT16_MAX, "deviceName length might not fit in a uint8_t");

Previously we had this check on the size , is this safe to not check mDeviceNameLength
I don't know the detail of this class just asking if we prevent it somehow..

Reply:
We add in the scan response array the following the [data Lenght] + [Data Type flag (0x9 for the name)] + [string]. This is the format for BLE device name

When doing an active scan you will receive the scan response (in opposition to the passive scan) and can decode the name this way.